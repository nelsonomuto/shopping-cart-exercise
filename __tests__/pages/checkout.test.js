import { shallow } from "enzyme"
import React from "react"
import CheckoutPage from "../../src/pages/checkout"

describe("CheckoutPage", () => {
  it("renders checkout page component", () => {
    const component = shallow(<CheckoutPage />)
    expect(component).toMatchSnapshot()
  })
})
