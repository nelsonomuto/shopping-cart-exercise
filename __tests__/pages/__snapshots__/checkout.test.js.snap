// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckoutPage renders checkout page component 1`] = `
<CartStoreProvider>
  <Layout>
    <SEO
      description=""
      keywords={Array []}
      lang="en"
      meta={Array []}
      title="Checkout"
    />
    <h1>
      Checkout
    </h1>
    <CheckoutCart />
  </Layout>
</CartStoreProvider>
`;
